examples/register_new_sensors_and_measures.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any
+from gym import spaces
+
+import numpy as np
+
+import habitat
+from habitat.config import Config as CN
+
+# Define the measure and register it with habitat
+# By default, the things are registered with the class name
+@habitat.registry.register_measure
+class EpisodeInfo(habitat.Measure):
+    def __init__(self, sim, config):
+        # This measure only needs the config
+        self._config = config
+
+        super().__init__()
+
+    # Defines the name of the measure in the measurements dictionary
+    def _get_uuid(self, *args: Any, **kwargs: Any):
+        return "episode_info"
+
+    # This is called whenver the environment is reset
+    def reset_metric(self, episode):
+        # Our measure always contains all the attributes of the episode
+        self._metric = vars(episode).copy()
+        # But only on reset, it has an additional field of my_value
+        self._metric["my_value"] = self._config.VALUE
+
+    # This is called whenver an action is taken in the environment
+    def update_metric(self, episode, action):
+        # Now the measure will just have all the attributes of the episode
+        self._metric = vars(episode).copy()
```

Comment:
Not sure, why you need to update `self._metric` each step, value doesn't change.

Reply:
It's a demo.  I am just showing how to add a new metric and what the difference between reset and update by setting it at every time-step and doing the `my_value` thing.